README.md

```diff
@@ -0,0 +1,44 @@
+skitter_api is a small wrapper around a set of [FlareSolverr](https://github.com/FlareSolverr/FlareSolverr) instances for load
+balancing and redundancy purposes. Essentially a series of docker containers
+comprised of gluetun providing a VPN connection and FlareSolverr providing CF
+workarounds are kept alive and proxied to.
+
+Currently these workers choose from a shared queue table stored in postgres;
+this is for ease-of-adapting from a previous project that was trivially
+shardable. This is far from ideal for a number of reasons. There's an initial
+effort to move to rabbitmq queues instead.
+
+Ensure you create a valid `etc/nginx/auth/skitter.htpasswd` file.
+
+`/etc/hosts` on the deploy target should contain an entry for polaris.fanfic.dev
+
+
+Create the docker container `docker/config.tsv`; see `docker/config.ex.tsv` for an
+example. The general format is:
+	`user_secret_name,pass_secret_name,subdomain,region`
```

Comment:
On the other hand there probably should be an extra blank line here, since the general format runs into the description. Maybe set it off with `csv` formatting explicitly?

Reply:
Something like this?:

Create the docker container `docker/config.tsv`; see `docker/config.ex.tsv` for an
example. 

The general format is:	
```csv
user_secret_name,pass_secret_name,subdomain,region
```

Reply:
Yup, precisely.